### ISSUE TITLE
optimizing hyper-parameters

### ISSUE THREAD

**Issue by Khayatian** [architecture-building-systems/CityEnergyAnalyst#862]
optimizing hyper parameters of the neural net:
layers, hidden nodes, transfer functions, learning rate, SGD algorithm
